src/cascadia/TerminalApp/Pane.h

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+// Module Name:
+// - Pane.h
+//
+// Abstract:
+// - Panes are an abstraction by which the terminal can dislay multiple terminal
+//   instances simultaneously in a single terminal window. While tabs allow for
+//   a single terminal window to have many terminal sessions running
+//   simultaneously within a single window, only one tab can be visible at a
+//   time. Panes, on the other hand, allow a user to have many different
+//   terminal sessions visible to the user within the context of a single window
+//   at the same time. This can enable greater productivity from the user, as
+//   they can see the output of one terminal window while working in another.
+// - See doc/cascadia/Panes.md for a detailed description.
+//
+// Author:
+// - Mike Griese (zadjii-msft) 16-May-2019
+
+
+#pragma once
+#include <winrt/Microsoft.Terminal.TerminalControl.h>
+#include "../../cascadia/inc/cppwinrt_utils.h"
+
+class Pane : public std::enable_shared_from_this<Pane>
+{
+
+public:
+
+    enum class SplitState : int
+    {
+        None = 0,
+        Vertical = 1,
+        Horizontal = 2
+    };
+
+    Pane(GUID profile, winrt::Microsoft::Terminal::TerminalControl::TermControl control, const bool lastFocused = false);
+    ~Pane() = default;

```

Comment:
@zadjii-msft nit:Since this class doesn't need a destructor it should be defined.
[C.20: If you can avoid defining default operations, do](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c20-if-you-can-avoid-defining-default-operations-do)